### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by FuturisticGoo**
Fixed some typos in the documentation

---

- [x] I’ve reviewed the contributor guide and applied the relevant portions to this PR.

<details>
  <summary>Contribution guidelines:</summary><br>

- See our [contributor guide](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md) for general expectations for PRs.
- Larger or significant changes should be discussed in an issue before creating a PR.
- Contributions to our repos should follow the [Dart style guide](https://dart.dev/guides/language/effective-dart) and use `dart format`.
- Most changes should add an entry to the changelog and may need to [rev the pubspec package version](https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change).
- Changes to packages require [corresponding tests](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md#Testing).

Note that many Dart repos have a weekly cadence for reviewing PRs - please allow for some latency before initial review feedback.
</details>

### CODE REVIEW

**Comment by FuturisticGoo** [comment]
image
I'm getting this error while trying to agree to the CLA

**Comment by natebosch** [comment]
> I'm getting this error while trying to agree to the CLA

@devoncarew - do you know who to contact about the CLA infrastructure?

@FuturisticGoo - it might be worth trying again after a while to see if it is a transient issue.

**Comment by FuturisticGoo** [comment]
Unfortunately still getting the same server error. I wouldn't mind anyone else incorporating this into their pull request, as it's just a typo fix, so feel free.

**Comment by FuturisticGoo** [comment]
It's fixed now, I got the CLA agreed.

**Comment by kevmoo** [comment]
Looks like you need to fix a long line!

File: lib/src/result/result.dart [hunk 1]
```diff
@@ -17,7 +17,7 @@ import 'value.dart';
 /// Capturing a result (either a returned value or a thrown error) means
 /// converting it into a [Result] - either a [ValueResult] or an [ErrorResult].
 ///
-/// This value can release itself by writing itself either to a [EventSink] or a
+/// This value can release itself by writing itself either to an [EventSink] or a
```
Comment (kevmoo) [inline lib/src/result/result.dart:20]: this line is now long, I think

**Review by kevmoo — APPROVED** [review]

**Pull request merged**